frontend/src/components/Metrics/MetricsStepper/index.tsx

```diff
@@ -209,6 +226,35 @@ const MetricsStepper = () => {
     setIsDialogShowing(false)
   }
 
+  const cleanPipelineToolState = () => {
```

Comment:
rename `cleanPipelineToolState` to `cleanPipelineToolConfiguration`

Reply:
> rename `cleanPipelineToolState` to `cleanPipelineToolConfiguration`

nice catch!